code/modules/food/kitchen/cooking_machines/container.dm

```diff
@@ -158,6 +163,23 @@
 			if (weights[I])
 				holder.trans_to(I, weights[I] / total)
 
+/obj/item/weapon/reagent_containers/cooking_container/update_icon()
+	overlays.Cut()
+
+	if(food_items)
+		var/image/filling = image('icons/obj/cooking_machines.dmi', src, "[icon_state]10")
+
+		var/percent = round((food_items / max_space) * 100)
+		switch(percent)
+			if(0 to 2)		filling.icon_state = "[icon_state]"
+			if(3 to 24) 	filling.icon_state = "[icon_state]1"
+			if(25 to 49)	filling.icon_state = "[icon_state]2"
+			if(50 to 74)	filling.icon_state = "[icon_state]3"
+			if(75 to 79)	filling.icon_state = "[icon_state]4"
+			if(80 to INFINITY)	filling.icon_state = "[icon_state]5"

```

Comment:
Nit: whitespace padding after the first non-whitespace character should use actual spaces so that the formatting is preserved across browser indent levels.